### DISCUSSION TITLE
You made my day!!!

### DISCUSSION THREAD

**Discussion by stefanbode** [SukramJ/aiohomematic#1067] [Show and tell]
Thank you all for this fantastic integration. I have half/half devices with TASMOTA and for the small ones homematic. The very bad integration of homematic in home assistant keep me on the old openHAB. Now I can finally move. Looking forward to contribute as I move forward with the migration into HAAS.

Is there any Discord server set-up  ?

**Comment by SukramJ** [comment]
Thanks for feedback, There is no Discord server in use.
